#!/usr/bin/env python
PACKAGE = "sr_point_cloud"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("use_convex_hull", bool_t, 0, "Remove planes before extracting clusters.", True)
gen.add("cluster_tolerance", double_t, 0, "Cluster tolerance", 0.05, 0.0, 100.0)
gen.add("min_cluster_size", int_t, 0, "Min points in a cluster.", 50, 0, 100)
gen.add("max_cluster_size", int_t, 0, "Max points in a cluster.", 25000, 0, 100)

exit(gen.generate(PACKAGE, "cluster_segmentor", "ClusterSegmentor"))

